docs/geedocs/answer/7160000.html

```diff
@@ -148,10 +149,95 @@
           <td>Close the resource form and open it again to make the save option available again.</td>
         </tr>
         <tr>
+          <td>340</td>
+          <td>GE Fusion Terrain is black</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
           <td>342</td>
           <td>Fusion crashes when opening an unsupported file type</td>
           <td>Re-open fusion and avoid opening unsupported file types.</td>
         </tr>
+        <tr>
+          <td>375</td>
+          <td>unbundle psycopg2</td>
+          <td>On Ubuntu 16.04 switch to python-psycopg2' instead of 'python2.7-psycopg2'.</td>
```

Comment:
Missing tick mark before python-psycopg2.

Also, I think it would be good to make the title more descriptive: maybe "Invalid version of psycopg2 on Ubuntu 16.04".

Reply:
ok for the '.' 
about the title, same question as before. We want to edit the title here but keep it as it is in the PR ? (375). The  PR title seems to point to a solution and not necessarily the issue.

Reply:
It currently says `On Ubuntu 16.04 switch to python-psycopg2'`.  It should say `On Ubuntu 16.04 switch to 'python-psycopg2'`.  The only change is to add an apostrophe.

The reason I'd suggest changing the title is that "unbundle psycopg2" sounds like an enhancement, but there is a known issue where some things don't work on Ubuntu 16.04 with the bundled psycopg2.  Maybe we should change the title here and on the issue.

Reply:
Added the missing '.
Changed the title in the release notes and added a comment/question in the 375 PR to eventually change its title.